app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -44,6 +48,22 @@ class MainActivity : AppCompatActivity() {
             handleNavigationVisibility(currentFragmentId)
         }
         handleAppLinkIntent(intent)
+        if (BuildConfig.FLAVOR == PLAY_STORE_BUILD_FLAVOR)
+            checkForUpdates()
+    }
+
+    private fun checkForUpdates() {
+        val appUpdateManager = AppUpdateManagerFactory.create(this)
+        val appUpdateInfoTask = appUpdateManager.appUpdateInfo
+        appUpdateInfoTask.addOnSuccessListener { appUpdateInfo ->
+            if (appUpdateInfo.updateAvailability() == UpdateAvailability.UPDATE_AVAILABLE) {
+                appUpdateManager.startUpdateFlowForResult(
+                    appUpdateInfo,
+                    AppUpdateType.IMMEDIATE,
```

Comment:
This should be a `FLEXIBLE` update instead. `IMMEDIATE` updates are done only when there are some major crashes in the app.